packages/nimble-components/src/table/components/cell/template.ts

```diff
@@ -20,10 +20,10 @@ export const template = html<TableCell>`
                 @toggle="${(x, c) => x.onActionMenuToggle(c.event as CustomEvent<MenuButtonToggleEventDetail>)}"
                 @click="${(_, c) => c.event.stopPropagation()}"
                 class="action-menu"
-                title="${x => x.actionMenuLabel ?? tableCellActionMenuLabel.getValueFor(x)}"
+                title="${x => (x.actionMenuLabel ? x.actionMenuLabel : tableCellActionMenuLabel.getValueFor(x))}"
```

Comment:
Is this an expected change in behavior? Nullish checks just undefined / null but truthiness also checks empty string. Might be a good change, dunno why one would want an empty string label..

Reply:
It is an intentional behavior change, but I can also write this a different way to avoid the behavior change (it would just be more verbose). I didn't see a reason why someone should create an empty label to override the default label, so I opted for making the logic simpler.

Reply:
makes sense! 🚢-it